anchors.php

```diff
@@ -70,7 +70,39 @@ public function onTwigSiteVariables()
                                     $class
                                     $truncate
                                 };
+                                // LO Hack: allow content to be hidden from the anchor 
```

Comment:
Please put this in the README.md, not in the comments in the code.  Also people don't like to see the word "hack" as it has negative connotations :)